uber/templates/registration/form.html

```diff
@@ -146,6 +146,27 @@
         </div>
     </div>
 
+    <div class="form-group">
+        <label for="admin_status" class="col-sm-2 control-label">Admin</label>
+        <div class="col-sm-6">
+            {% if attendee.admin_account %}
+                <p name="admin_status">True</p>
+            {% else %}
+                <p name="admin_status">False</p>
+            {% endif %}
+        </div>
+    </div>
+
+    {% if attendee.admin_account %}
+        <div class="form-group">
+            <label for="admin_options" class="col-sm-2 control-label">Permissions</label>
+            <div class="col-sm-6">
+                {% checkgroup attendee.admin_account.access %}
```

Comment:
A ``checkgroup`` is probably wrong here, because checking or unchecking those checkboxes isn't going to actually do anything, whereas someone who saw the checkboxes would probably assume that they could check or uncheck them and have those things saved.